docs/src/main/sphinx/release/release-456.md

```diff
@@ -0,0 +1,86 @@
+# Release 456 (5 Sep 2024)
+
+## General
+
+* Expose additional JMX metrics about resource groups, including CPU and memory
+  usage, limits, and scheduling policy. ({issue}`22957`)
+* Add [adaptive plan optimization](/optimizer/adaptive-plan-optimizations) which
+  can reorder joins at runtime. The feature can be disabled with the
+  `fault-tolerant-execution-adaptive-join-reordering-enabled` configuration
+  property or the `fault_tolerant_execution_adaptive_join_reordering_enabled`
+  session property. ({issue}`23046`)
+* Improve performance for [file compression and decompression](file-compression)
+  of ORC and Parquet files used for [exchange spooling with fault-tolerant
+  execution mode](fte-exchange-manager). ({issue}`22532`)
+* Extend the Kafka event listener to send split completion events. ({issue}`23065`)
+
+## JDBC driver
+
+* Publishing a JDBC driver JAR without bundled, third-party dependencies. ({issue}`22098`)
```

Comment:
```suggestion
* A new JDBC driver JAR without bundled, third-party dependencies is now available as <original-jar-name>-non-shaded. ({issue}`22098`)
```

(please fill in original jar name, I don't remember what it's called on Maven central).

Reply:
We dont need that level of detail in the release notes. I already updated the docs and will just link to it here.